### ISSUE TITLE
Problem with libjpeg?

### ISSUE THREAD

**Issue by ngam** [conda-forge/torchvision-feedstock#67]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

```
>>> import torchvision
/Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/io/image.py:13: UserWarning: Failed to load image Python extension: dlopen(/Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/image.so, 0x0006): Library not loaded: @rpath/libjpeg.8.dylib
  Referenced from: <BE0CCD9A-269A-30E2-A23C-DA45E89EBB1F> /Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/image.so
  Reason: tried: '/Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/../../../libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/../../../libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/lib/python3.10/lib-dynload/../../libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/bin/../lib/libjpeg.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS@rpath/libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/../../../libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/lib/python3.10/site-packages/torchvision/../../../libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/lib/python3.10/lib-dynload/../../libjpeg.8.dylib' (no such file), '/Users/ngam/.micromamba/envs/test/bin/../lib/libjpeg.8.dylib' (no such file), '/usr/local/lib/libjpeg.8.dylib' (no such file), '/usr/lib/libjpeg.8.dylib' (no such file, not in dyld cache)
  warn(f"Failed to load image Python extension: {e}")
>>>
>>> torchvision.__version__
'0.14.1a0'
>>> 

```

From https://anaconda.org/conda-forge/torchvision/files: 

<h4 class="modal-title" id="myModalLabel" style="font-style: normal; font-variant-caps: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; margin: 0.2rem 0px 0.5rem; padding: 0px; color: rgb(34, 34, 34); font-family: &quot;Museo Sans Rounded&quot;, Lato, &quot;Helvetica Neue&quot;, Helvetica, Roboto, Arial, sans-serif; font-weight: 400; line-height: 1.4; text-rendering: optimizeLegibility; font-size: 1.4375rem; caret-color: rgb(34, 34, 34);">osx-arm64/torchvision-0.14.1-cpu_py38h3c81145_0.conda</h4><p class="subheader" style="font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; margin: 0.2rem 0px 0.5rem; padding: 0px; font-family: &quot;Museo Sans Rounded&quot;, Lato, &quot;Helvetica Neue&quot;, Helvetica, Roboto, Arial, sans-serif; text-rendering: optimizeLegibility; line-height: 1.4; color: rgb(111, 111, 111);">No Description</p><div style="font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; margin: 0px; padding: 0px; caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Museo Sans Rounded&quot;, Lato, &quot;Helvetica Neue&quot;, Helvetica, Roboto, Arial, sans-serif;">

Uploaded | Sat Jan 21 19:09:31 2023
-- | --
md5 checksum | b31801a7dbd49a06728b098cef10d692
arch | arm64
build | cpu_py38h3c81145_0
constrains | pytorch =*=cpu*
depends | jpeg >=9e,<10a, libcxx >=14.0.6, libpng >=1.6.39,<1.7.0a0, numpy >=1.20.3,<2.0a0, pillow >=5.3.0,!=8.3.0,!=8.3.1, python >=3.8,<3.9.0a0, python >=3.8,<3.9.0a0 *_cpython, python_abi 3.8.* *_cp38, pytorch >=1.13.1,<1.14.0a0, requests
license | BSD-3-Clause
license_family | BSD
machine | arm64
operatingsystem | darwin
platform | osx
subdir | osx-arm64
target-triplet | arm64-any-darwin
timestamp | 1674328148821

</div>

### Installed packages

```shell
(test) ~$ micromamba list
List of packages in environment: "/Users/ngam/.micromamba/envs/test"

  Name                           Version      Build                 Channel    
─────────────────────────────────────────────────────────────────────────────────
  aiofiles                       22.1.0       pyhd8ed1ab_0          conda-forge
  aiosqlite                      0.18.0       pyhd8ed1ab_0          conda-forge
  anyio                          3.6.2        pyhd8ed1ab_0          conda-forge
  appnope                        0.1.3        pyhd8ed1ab_0          conda-forge
  argon2-cffi                    21.3.0       pyhd8ed1ab_0          conda-forge
  argon2-cffi-bindings           21.2.0       py310h8e9501a_3       conda-forge
  asttokens                      2.2.1        pyhd8ed1ab_0          conda-forge
  attrs                          22.2.0       pyh71513ae_0          conda-forge
  babel                          2.11.0       pyhd8ed1ab_0          conda-forge
  backcall                       0.2.0        pyh9f0ad1d_0          conda-forge
  backports                      1.0          pyhd8ed1ab_3          conda-forge
  backports.functools_lru_cache  1.6.4        pyhd8ed1ab_0          conda-forge
  beautifulsoup4                 4.11.2       pyha770c72_0          conda-forge
  bleach                         6.0.0        pyhd8ed1ab_0          conda-forge
  brotlipy                       0.7.0        py310h8e9501a_1005    conda-forge
  bzip2                          1.0.8        h3422bc3_4            conda-forge
  ca-certificates                2022.12.7    h4653dfc_0            conda-forge
  certifi                        2022.12.7    pyhd8ed1ab_0          conda-forge
  cffi                           1.15.1       py310h2399d43_3       conda-forge
  charset-normalizer             2.1.1        pyhd8ed1ab_0          conda-forge
  comm                           0.1.2        pyhd8ed1ab_0          conda-forge
  cryptography                   39.0.1       py310hfc83b78_0       conda-forge
  debugpy                        1.6.6        py310h0f1eb42_0       conda-forge
  decorator                      5.1.1        pyhd8ed1ab_0          conda-forge
  defusedxml                     0.7.1        pyhd8ed1ab_0          conda-forge
  entrypoints                    0.4          pyhd8ed1ab_0          conda-forge
  executing                      1.2.0        pyhd8ed1ab_0          conda-forge
  flit-core                      3.8.0        pyhd8ed1ab_0          conda-forge
  freetype                       2.12.1       hd633e50_1            conda-forge
  idna                           3.4          pyhd8ed1ab_0          conda-forge
  importlib-metadata             6.0.0        pyha770c72_0          conda-forge
  importlib_metadata             6.0.0        hd8ed1ab_0            conda-forge
  importlib_resources            5.10.2       pyhd8ed1ab_0          conda-forge
  ipykernel                      6.21.1       pyh736e0ef_0          conda-forge
  ipython                        8.10.0       pyhd1c38e8_0          conda-forge
  ipython_genutils               0.2.0        py_1                  conda-forge
  jedi                           0.18.2       pyhd8ed1ab_0          conda-forge
  jinja2                         3.1.2        pyhd8ed1ab_1          conda-forge
  jpeg                           9e           h1a8c8d9_3            conda-forge
  json5                          0.9.5        pyh9f0ad1d_0          conda-forge
  jsonschema                     4.17.3       pyhd8ed1ab_0          conda-forge
  jupyter_client                 8.0.2        pyhd8ed1ab_0          conda-forge
  jupyter_core                   5.2.0        py310hbe9552e_0       conda-forge
  jupyter_events                 0.5.0        pyhd8ed1ab_1          conda-forge
  jupyter_server                 2.2.1        pyhd8ed1ab_0          conda-forge
  jupyter_server_fileid          0.6.0        pyhd8ed1ab_0          conda-forge
  jupyter_server_terminals       0.4.4        pyhd8ed1ab_1          conda-forge
  jupyter_server_ydoc            0.6.1        pyhd8ed1ab_0          conda-forge
  jupyter_ydoc                   0.2.2        pyhd8ed1ab_0          conda-forge
  jupyterlab                     3.6.1        pyhd8ed1ab_0          conda-forge
  jupyterlab_pygments            0.2.2        pyhd8ed1ab_0          conda-forge
  jupyterlab_server              2.19.0       pyhd8ed1ab_0          conda-forge
  lcms2                          2.14         h481adae_1            conda-forge
  lerc                           4.0.0        h9a09cb3_0            conda-forge
  libblas                        3.9.0        16_osxarm64_openblas  conda-forge
  libcblas                       3.9.0        16_osxarm64_openblas  conda-forge
  libcxx                         14.0.6       h2692d47_0            conda-forge
  libdeflate                     1.17         h1a8c8d9_0            conda-forge
  libffi                         3.4.2        h3422bc3_5            conda-forge
  libgfortran                    5.0.0        11_3_0_hd922786_27    conda-forge
  libgfortran5                   11.3.0       hdaf2cc0_27           conda-forge
  liblapack                      3.9.0        16_osxarm64_openblas  conda-forge
  libopenblas                    0.3.21       openmp_hc731615_3     conda-forge
  libpng                         1.6.39       h76d750c_0            conda-forge
  libprotobuf                    3.21.12      hb5ab8b9_0            conda-forge
  libsodium                      1.0.18       h27ca646_1            conda-forge
  libsqlite                      3.40.0       h76d750c_0            conda-forge
  libtiff                        4.5.0        h5dffbdd_2            conda-forge
  libwebp-base                   1.2.4        h57fd34a_0            conda-forge
  libxcb                         1.13         h9b22ae9_1004         conda-forge
  libzlib                        1.2.13       h03a7124_4            conda-forge
  llvm-openmp                    15.0.7       h7cfbb63_0            conda-forge
  markupsafe                     2.1.2        py310h8e9501a_0       conda-forge
  matplotlib-inline              0.1.6        pyhd8ed1ab_0          conda-forge
  mistune                        2.0.5        pyhd8ed1ab_0          conda-forge
  nbclassic                      0.5.1        pyhd8ed1ab_0          conda-forge
  nbclient                       0.7.2        pyhd8ed1ab_0          conda-forge
  nbconvert                      7.2.9        pyhd8ed1ab_0          conda-forge
  nbconvert-core                 7.2.9        pyhd8ed1ab_0          conda-forge
  nbconvert-pandoc               7.2.9        pyhd8ed1ab_0          conda-forge
  nbformat                       5.7.3        pyhd8ed1ab_0          conda-forge
  ncurses                        6.3          h07bb92c_1            conda-forge
  nest-asyncio                   1.5.6        pyhd8ed1ab_0          conda-forge
  notebook                       6.5.2        pyha770c72_1          conda-forge
  notebook-shim                  0.2.2        pyhd8ed1ab_0          conda-forge
  numpy                          1.24.2       py310h3d2048e_0       conda-forge
  openjpeg                       2.5.0        hbc2ba62_2            conda-forge
  openssl                        3.0.8        h03a7124_0            conda-forge
  packaging                      23.0         pyhd8ed1ab_0          conda-forge
  pandas                         1.5.3        py310h2b830bf_0       conda-forge
  pandoc                         2.19.2       hce30654_1            conda-forge
  pandocfilters                  1.5.0        pyhd8ed1ab_0          conda-forge
  parso                          0.8.3        pyhd8ed1ab_0          conda-forge
  pexpect                        4.8.0        pyh1a96a4e_2          conda-forge
  pickleshare                    0.7.5        py_1003               conda-forge
  pillow                         9.4.0        py310h5a7539a_1       conda-forge
  pip                            23.0         pyhd8ed1ab_0          conda-forge
  pkgutil-resolve-name           1.3.10       pyhd8ed1ab_0          conda-forge
  platformdirs                   3.0.0        pyhd8ed1ab_0          conda-forge
  prometheus_client              0.16.0       pyhd8ed1ab_0          conda-forge
  prompt-toolkit                 3.0.36       pyha770c72_0          conda-forge
  psutil                         5.9.4        py310h8e9501a_0       conda-forge
  pthread-stubs                  0.4          h27ca646_1001         conda-forge
  ptyprocess                     0.7.0        pyhd3deb0d_0          conda-forge
  pure_eval                      0.2.2        pyhd8ed1ab_0          conda-forge
  pycparser                      2.21         pyhd8ed1ab_0          conda-forge
  pygments                       2.14.0       pyhd8ed1ab_0          conda-forge
  pyopenssl                      23.0.0       pyhd8ed1ab_0          conda-forge
  pyrsistent                     0.19.3       py310h8e9501a_0       conda-forge
  pysocks                        1.7.1        pyha2e5f31_6          conda-forge
  python                         3.10.9       h3ba56d0_0_cpython    conda-forge
  python-dateutil                2.8.2        pyhd8ed1ab_0          conda-forge
  python-fastjsonschema          2.16.2       pyhd8ed1ab_0          conda-forge
  python-json-logger             2.0.4        pyhd8ed1ab_0          conda-forge
  python_abi                     3.10         3_cp310               conda-forge
  pytorch                        1.13.1       cpu_py310h32bc11d_1   conda-forge
  pytz                           2022.7.1     pyhd8ed1ab_0          conda-forge
  pyyaml                         6.0          py310h8e9501a_5       conda-forge
  pyzmq                          25.0.0       py310hc407298_0       conda-forge
  readline                       8.1.2        h46ed386_0            conda-forge
  requests                       2.28.2       pyhd8ed1ab_0          conda-forge
  send2trash                     1.8.0        pyhd8ed1ab_0          conda-forge
  setuptools                     67.1.0       pyhd8ed1ab_0          conda-forge
  six                            1.16.0       pyh6c4a22f_0          conda-forge
  sleef                          3.5.1        h156473d_2            conda-forge
  sniffio                        1.3.0        pyhd8ed1ab_0          conda-forge
  soupsieve                      2.3.2.post1  pyhd8ed1ab_0          conda-forge
  stack_data                     0.6.2        pyhd8ed1ab_0          conda-forge
  terminado                      0.17.1       pyhd1c38e8_0          conda-forge
  tinycss2                       1.2.1        pyhd8ed1ab_0          conda-forge
  tk                             8.6.12       he1e0b03_0            conda-forge
  tomli                          2.0.1        pyhd8ed1ab_0          conda-forge
  torchvision                    0.14.1       cpu_py310h7df52bc_0   conda-forge
  tornado                        6.2          py310h8e9501a_1       conda-forge
  traitlets                      5.9.0        pyhd8ed1ab_0          conda-forge
  typing-extensions              4.4.0        hd8ed1ab_0            conda-forge
  typing_extensions              4.4.0        pyha770c72_0          conda-forge
  tzdata                         2022g        h191b570_0            conda-forge
  urllib3                        1.26.14      pyhd8ed1ab_0          conda-forge
  wcwidth                        0.2.6        pyhd8ed1ab_0          conda-forge
  webencodings                   0.5.1        py_1                  conda-forge
  websocket-client               1.5.1        pyhd8ed1ab_0          conda-forge
  wheel                          0.38.4       pyhd8ed1ab_0          conda-forge
  xorg-libxau                    1.0.9        h27ca646_0            conda-forge
  xorg-libxdmcp                  1.1.3        h27ca646_0            conda-forge
  xz                             5.2.6        h57fd34a_0            conda-forge
  y-py                           0.5.5        py310h1dc5ec9_2       conda-forge
  yaml                           0.2.5        h3422bc3_2            conda-forge
  ypy-websocket                  0.8.2        pyhd8ed1ab_0          conda-forge
  zeromq                         4.3.4        hbdafb3b_1            conda-forge
  zipp                           3.13.0       pyhd8ed1ab_0          conda-forge
  zstd                           1.5.2        hf913c23_6            conda-forge
(test) ~$
```

### Environment info

```shell
(test) ~$ micromamba info

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/


            environment : test (active)
           env location : /Users/ngam/.micromamba/envs/test
      user config files : /Users/ngam/.mambarc
 populated config files : /Users/ngam/.condarc
       libmamba version : 1.3.1
     micromamba version : 1.3.1
           curl version : libcurl/7.86.0 SecureTransport (OpenSSL/3.0.8) zlib/1.2.13 libssh2/1.10.0 nghttp2/1.47.0
     libarchive version : libarchive 3.6.2 zlib/1.2.13 bz2lib/1.0.8 libzstd/1.5.2
       virtual packages : __unix=0=0
                          __osx=13.2=0
                          __archspec=1=arm64
               channels : https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
       base environment : /Users/ngam/.micromamba
               platform : osx-arm64
(test) ~$
```

**Comment by hmaarrfk** [comment]
This is because oyttorch accidentally found libjpegtirbo from pillow during the build.


This feedstock likely should be marked as broken for Thai latest build and rerun.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
No. This was a big that we introduced by not having constraints that excluded the two versions of jpeg. 

See discussion and referenced and cross referenced issue in 
https://github.com/conda-forge/conda-forge.github.io/issues/673

The TLDR is that hopefully we move to turbo soon.
